load/UpdateDBPTextFilesets.py

```diff
@@ -72,7 +72,8 @@ def validateFileset(self, subTypeCode, bibleId, filesetId, languageRecord, lptsI
 			cmd = [self.config.node_exe,
 				self.config.sofria_client_js,
 				fullFilesetPath,
-				self.config.directory_accepted]
+				self.config.directory_accepted,
+				lptsDamId[:7] + "_" + lptsDamId[8:]]
```

Comment:
@bradflood I have done a change into sofria-client to sent the folder name where json files will be stored as parameter.

You can see in the following URL  sofria-client PR:
- https://github.com/faithcomesbyhearing/sofria-cli/pull/5

Reply:
there should be only one place where this type of logic occurs
`lptsDamId[:7] + "_" + lptsDamId[8:]]`
If this is the only place, that is fine. But if we are doing it elsewhere, make sure we store the computed value in use it everywhere else

Reply:
@bradflood , you are right.
I have created a new method called:  `filesetLptsDamName` to wrap this logic. You can see it in the following URL.
https://github.com/faithcomesbyhearing/dbp-etl/pull/72/files#diff-160bcec181f4365fc933594ec0b0356893dc185d2f4d9fc9e8a81f732eb3ef15R228